topics/galaxy-interface/tutorials/upload-rules-advanced/tutorial.md

```diff
@@ -93,7 +93,7 @@ We will use this information to build a collection of FASTA files from our query
 >      ```
 >
 >    - Click `Build`
->   ![screenshot](../../images/rules/rules_example_4_2_initial_rules.png)
+>   ![screenshot](http://galaxy-tests.s3.amazonaws.com/galaxy-gtn-screenshots/local/rules_example_4_2_initial_rules.png)
```

Comment:
The GTN is served with HTTPS only, and some browsers will throw an error for mixed content like this, I think we might need to find a solution there. (It could be as simple as proxying it, like we do in some other cases https://github.com/galaxyproject/infrastructure-playbook.)

Additionally it's unfortunate that this will now require an internet connection just to view the tutorial, but I definitely understand the ease of doing it this way. Maybe we can find some alternative solution in the future there.

Reply:
Ah, that easy, cool :)

Reply:
>Ah, that easy, cool :)

Yeah :)

> Additionally it's unfortunate that this will now require an internet connection just to view the tutorial, but I definitely understand the ease of doing it this way. Maybe we can find some alternative solution in the future there.

I mean, it's a website. You still need internet to access it. I guess, it might be a problem only if you want to deploy it locally and you don't have internet for some reason.

Reply:
Yeah, exactly, for internet-restricted options. Also e.g. when you're working on the ICE and the internet is incredibly unreliable, those situations it's nice that you can work completely offline. But we can address that issue later! This looks good for now and as soon as someone confirms the images are sufficiently similar we can merge this :)